agent/lib/kontena/workers/node_info_worker.rb

```diff
@@ -158,17 +164,28 @@ def send_statsd_metrics(event)
       statsd.gauge("#{key_base}.cpu.load.1m", event[:load][:'1m'])
       statsd.gauge("#{key_base}.cpu.load.5m", event[:load][:'5m'])
       statsd.gauge("#{key_base}.cpu.load.15m", event[:load][:'15m'])
+      statsd.gauge("#{key_base}.cpu.system", event[:cpu][:system])
+      statsd.gauge("#{key_base}.cpu.user", event[:cpu][:user])
+      statsd.gauge("#{key_base}.cpu.idle", event[:cpu][:idle])
       statsd.gauge("#{key_base}.memory.active", event[:memory][:active])
       statsd.gauge("#{key_base}.memory.free", event[:memory][:free])
       statsd.gauge("#{key_base}.memory.total", event[:memory][:total])
       statsd.gauge("#{key_base}.usage.container_seconds", event[:usage][:container_seconds])
       event[:filesystem].each do |fs|
-        name = fs[:name].split("/")[1..-1].join(".")
+        name = fs[:name] ? fs[:name].split("/")[1..-1].join(".") : ""
         statsd.gauge("#{key_base}.filesystem.#{name}.free", fs[:free])
         statsd.gauge("#{key_base}.filesystem.#{name}.available", fs[:available])
         statsd.gauge("#{key_base}.filesystem.#{name}.used", fs[:used])
         statsd.gauge("#{key_base}.filesystem.#{name}.total", fs[:total])
       end
+      event[:network].each do |network|
+        name = network[:name] ? network[:name] : ""
```

Comment:
The network interface names might also include `.`, for VLAN interfaces names like `eth0.8`. Escape those to something else to keep the statsd schema at a fixed number of levels?